locales/en/plugin__kubevirt-plugin.json

```diff
@@ -543,12 +543,12 @@
   "Guest agent is required": "Guest agent is required",
   "Guest login credentials": "Guest login credentials",
   "Guest management": "Guest management",
-  "Guest system access log": "Guest system access log",
-  "Guest system access logs are disabled at cluster": "Guest system access logs are disabled at cluster",
-  "Guest system access logs are disabled at VirtualMachine": "Guest system access logs are disabled at VirtualMachine",
-  "Guest system access logs disabled at cluster": "Guest system access logs disabled at cluster",
-  "Guest system access logs not ready. a restart required": "Guest system access logs not ready. a restart required",
+  "Guest system log": "Guest system log",
   "Guest system log access": "Guest system log access",
+  "Guest system logs are disabled at cluster": "Guest system logs are disabled at cluster",
```

Comment:
Well, this is a question for the doc team, not me. Also this PR is about shortening "Guest system access log" to "Guest system log", not improving the whole sentences, even when there is a lot to improve, of course. Unfortunately, Avital is no longer in the doc team and I have no idea who to ping instead. If you know who to ping, let's do that to improve the whole sentences.

Reply:
I didn't realize that Avital left. Yifat said she's waiting for her point of contact to give her the name of who we should contact instead. We can put these on the back burner. I'll lgtm it so it can be merged once the hold comes off.

Reply:
Thank you, yes, that's what I was told, too. Once we know, we can fix this by a followup PR for sure :) And the hold is here just to wait for Aviv's big PF PR merged first.

Reply:
@ousleyp can you please chime in on this ensuring consistency with our docs?

Reply:
My understanding (per @tiraboschi) is that the cluster-level setting is not globally disabling/enabling guest system logs, just setting the default for the cluster.

Does it make sense to have this message when it is possible for guest system logs to be enabled on a VM and still disabled by default on the cluster? It feels a little bit misleading. If I read that guest system logs are disabled on the cluster, I might think that I cannot enable them for the individual VM.

(Thoughts for future improvements, maybe.)

Reply:
Correct: we have a cluster wide default value (disabled as default in CNV 4.15) that is used if nothing is specifically set at VM level 
VM specific value (if there) always wins over the cluster wide default.
It can be off by default at cluster level but individually enabled for a specific VM.